### PR TITLE
Fix/missing tanslations

### LINKED ISSUE #2217: missing translation for string "Start of vacation" in admin 1.9.12 Auto-reply form
## Before I've opened the issue
- [x] Checked if no issue or pull-request for this already exists.
- [x] Checked [documentation](https://mailu.io/master/) and [FAQ](https://mailu.io/master/faq.html). (Tip, use the search function on the documentation page)
- [x] I've understood `Mailu` is made by volunteers in their **free time** — be conscise, civil and accept that delays can occur.
- [x] The title of the issue is short and simple but precise. It contains specific terms related to the actual issue. I was specific while writing the title.

### Environment
 - [x] docker-compose
 - [ ] kubernetes
 - [ ] docker swarm

### Versions
1.9.12. fresh docker-compose pull minutes before writing this issue.

## Description
In `/admin` there is a missing translation for the string "Start of vacation" in the "Auto-reply" dialog. At least for DE and ES the translation is missing.

## Replication Steps
Login into `/admin`, go to "Auto-reply" and change the language via the languages selector in the top right corner to "Deutsch".  See the string remains untranslated "Start of vacation".

## Expected behaviour
The string should be translated for DE as "Beginn der Abwesenheit".

### PR DESCRIPTION
**Pull request by DjVinnii**
## What type of PR?

Fix/Enhancement

## What does this PR do?
Add missing Dutch translation, as well as the German translation for `Start of vacation`

### Related issue(s)
- closes #2217

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [ ] In case of feature or enhancement: documentation updated accordingly
- [ ] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

File: core/admin/mailu/translations/nl/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -304,15 +304,15 @@ msgstr "Anti-spam"
 
 #: mailu/ui/templates/antispam.html:8
 msgid "RSPAMD status page"
-msgstr ""
+msgstr "RSPAMD stauts pagina"
```
Comment (Diman0) [inline core/admin/mailu/translations/nl/LC_MESSAGES/messages.po:307]: Small typo

**Review by Diman0 — CHANGES_REQUESTED** [review]
Thank you for your contribution. There is a small typo in one of the translations. Once that is addressed the PR looks good to me.

File: core/admin/mailu/translations/nl/LC_MESSAGES/messages.po (hunk 1 above)
Comment (DjVinnii) [reply core/admin/mailu/translations/nl/LC_MESSAGES/messages.po]: Whoops. I fixed it!

**Comment by DjVinnii** [comment]
I marked this PR as draft, because it might be nice to get some more German translations as mentioned in the original issue. If this takes too long, the PR can be merged.

**Comment by ghostwheel42** [comment]
@DjVinnii I've added the missing german translations.

**Comment by DjVinnii** [comment]
@ghostwheel42 Thanks! I'll mark the PR as Ready for review

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -305,15 +305,15 @@ msgstr "Antispam"
 
 #: mailu/ui/templates/antispam.html:8
 msgid "RSPAMD status page"
-msgstr ""
+msgstr "RSPAMD Statusseite"
 
 #: mailu/ui/templates/client.html:8
 msgid "configure your email client"
-msgstr ""
+msgstr "iInformationen zur Einrichtung Ihres Email-Clients"
```
Comment (level420) [inline core/admin/mailu/translations/de/LC_MESSAGES/messages.po:312]: The first charachter i is superfluous

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -305,15 +305,15 @@ msgstr "Antispam"
...
 
 #: mailu/ui/templates/client.html:13
 msgid "Incoming mail"
-msgstr ""
+msgstr "E-Mail-Empfang"
 
 #: mailu/ui/templates/client.html:16 mailu/ui/templates/client.html:41
 msgid "Mail protocol"
```
Comment (level420) [inline core/admin/mailu/translations/de/LC_MESSAGES/messages.po:319]: "E-Mail-Protokoll"

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -173,7 +173,7 @@ msgstr "Spamfilter aktivieren"
 
 #: mailu/ui/forms.py:103
 msgid "Enable marking spam mails as read"
-msgstr ""
+msgstr "Spam automatisch als gelesen markieren"
 
 #: mailu/ui/forms.py:104
 msgid "Spam filter tolerance"
```
Comment (level420) [inline core/admin/mailu/translations/de/LC_MESSAGES/messages.po:179]: Spam-Filter-Toleranz

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -218,7 +218,7 @@ msgstr "Text"
 
 #: mailu/ui/forms.py:122
 msgid "Start of vacation"
-msgstr ""
+msgstr "Beginn der Abwesenheit"
 
 #: mailu/ui/forms.py:123
 msgid "End of vacation"
```
Comment (level420) [inline core/admin/mailu/translations/de/LC_MESSAGES/messages.po:224]: "Ende der Abwesenheit"

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -325,7 +325,7 @@ msgstr "Servername"
 
 #: mailu/ui/templates/client.html:38
 msgid "Outgoing mail"
-msgstr ""
+msgstr "E-Mail-Versand"
 
 #: mailu/ui/templates/confirm.html:4
 msgid "Confirm action"
```
Comment (level420) [inline core/admin/mailu/translations/de/LC_MESSAGES/messages.po:331]: "Aktion bestätigen"

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -325,7 +325,7 @@ msgstr "Servername"
 
 #: mailu/ui/templates/client.html:38
 msgid "Outgoing mail"
-msgstr ""
+msgstr "E-Mail-Versand"
```
Comment (level420) [inline core/admin/mailu/translations/de/LC_MESSAGES/messages.po:329]: I'd rather simply write "Versendete E-Mail"

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -53,7 +53,7 @@ msgstr "Client Einrichtung"
 
 #: mailu/sso/templates/sidebar_sso.html:16 mailu/ui/templates/sidebar.html:114
 msgid "Website"
-msgstr "Website"
+msgstr "mailu.io"
```
Comment (level420) [inline core/admin/mailu/translations/de/LC_MESSAGES/messages.po:56]: Why not leave this as it is?

**Review by level420 — CHANGES_REQUESTED** [review]
Nearly perfect. Only one correction. See my comment.

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po (hunk 8 above)
Comment (ghostwheel42) [reply core/admin/mailu/translations/de/LC_MESSAGES/messages.po:56]: Yes, it's probably better to just write "Webseite" here. After all mailu.io can be configured AFAIK.

Comment (ghostwheel42) [reply core/admin/mailu/translations/de/LC_MESSAGES/messages.po]: changed back

File: core/admin/mailu/translations/de/LC_MESSAGES/messages.po (hunk 2 above)
Comment (ghostwheel42) [reply core/admin/mailu/translations/de/LC_MESSAGES/messages.po]: fixed.

File: core/admin/mailu/translations/nl/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -31,15 +31,15 @@ msgstr "Aanmelden"
 
 #: mailu/sso/templates/base_sso.html:8 mailu/ui/templates/base.html:8
 msgid "Admin page for"
-msgstr ""
+msgstr "Admin pagina voor"
 
 #: mailu/sso/templates/base_sso.html:19 mailu/ui/templates/base.html:19
 msgid "toggle sidebar"
 msgstr ""
```
Comment (Diman0) [inline core/admin/mailu/translations/nl/LC_MESSAGES/messages.po:38]: This is the last missing Dutch string.

**Review by Diman0 — DISMISSED** [review]
For Dutch there is one last missing string. This evening I can probably add this missing string.

Edit: I marked the wrong line number. It should be line number 38 of course.

**Review by level420 — APPROVED** [review]

**Comment by Diman0** [comment]
Translations should be complete now.

**Review by Diman0 — APPROVED** [review]
lgtm

**Review by ghostwheel42 — APPROVED** [review]
Lgtm

**Pull request merged**